Dockerfile

```diff
@@ -121,6 +121,10 @@ ADD . /src
 WORKDIR /src
 RUN pip install -e .
 
+# Setup git config used by github commit pushing.
```

Comment:
It is not a big deal here, but it would be wise to get in the habit of thinking a lot about what constitutes a checkpoint in Docker. These commands are making an additional image checkpoint _each_.

I would have expected this to be written:

```Dockerfile
RUN git config --global user.email googleapis-publisher@google.com \
  && git config --global user.name "Googleapis Publisher"
```

This makes one checkpoint instead of two.